go.mod

```diff
@@ -5,18 +5,20 @@ require (
 	github.com/go-logfmt/logfmt v0.4.0 // indirect
 	github.com/go-stack/stack v1.8.0 // indirect
 	github.com/gogo/protobuf v1.2.1
-	github.com/golang/protobuf v1.3.2
+	github.com/golang/protobuf v1.4.0
```

Comment:
Is this really a backwards compatible change? I know some signatures might change. Could you try running `gorelease` on this branch compared to `master`? See https://godoc.org/golang.org/x/exp/cmd/gorelease.

Reply:
I returned the protobuf to version 1.3.2 and ran gorelease, this was the result:

```
github.com/grpc-ecosystem/go-grpc-middleware/logging/logrus
-----------------------------------------------------------
Compatible changes:
- DefaultMessageProducer: added
- MessageProducer: added
- WithMessageProducer: added

github.com/grpc-ecosystem/go-grpc-middleware/logging/zap
--------------------------------------------------------
Compatible changes:
- DefaultMessageProducer: added
- MessageProducer: added
- WithMessageProducer: added

github.com/grpc-ecosystem/go-grpc-middleware/tracing/opentracing
----------------------------------------------------------------
Compatible changes:
- UnaryRequestHandlerFunc: added
- WithTraceHeaderName: added
- WithUnaryRequestHandlerFunc: added

github.com/grpc-ecosystem/go-grpc-middleware/validator
------------------------------------------------------
Compatible changes:
- UnaryClientInterceptor: added

Inferred base version: v1.2.0
Suggested version: v1.3.0
```

Reply:
Ah, yeah, no, I meant with version 1.4.0, but okay lets roll with 1.3.2 it will be less complicated.